src/checkout/views/Checkout/Checkout.tsx

```diff
@@ -0,0 +1,41 @@
+import React from "react";
+import { Container, useForm, Form, Link, useTranslate } from "../../../_shared";
+import { FormData } from "../../types";
+import { schemaCheckoutForm, defaultValuesForm } from "../../utils";
+import { FormOptions } from "../../components";
+import { Grid } from "../../../_shared";
+
+export const Checkout = () => {
+  const translate = useTranslate();
+
+  const formContextValue = useForm<FormData>({
```

Comment:
Maybe we can change the FormData name with something like CheckOutData or similar